kubernetes/api.yml

```diff
@@ -49,8 +49,8 @@ spec:
             memory: 250Mi
             cpu: 200m
           limits:
-            memory: 500Mi
-            cpu: 500m
+            memory: 1000Mi
```

Comment:
What's the reason for going back to `1000Mi` instead of `1400Mi`?

Reply:
For setting up requests and limits, my reference is still https://fr.sysdig.com/blog/kubernetes-limits-requests/
In terms of "limits", they say "Regarding limits, achieving a good setting is a matter of try and catch. There is no optimal value for everyone as it hardly depends on the nature of the application, the demand model, the tolerance to errors and many other factors."
So, as a rule of thumb, I look at the max CPU used by the container and then give it a bit of extra room for running that specific high demanding task, but not too much, so to "cap" the CPU utilisation in case of a runaway process.
In this specific case, I noticed the container **almost** hitting the 500m limit, so I thought that 1000m would give it enough room for the API update, but 1400m would have been over-provisioning. 
Anyway, as you can see it's all a wild guess here, so happy to review later or if you link me to documentation that can explain better the criteria for setting the limits.